net.c

```diff
@@ -88,6 +89,8 @@ static void nlpkt_close(struct nlpkt *pkt)
 	free(pkt->data);
 }
 
+#pragma GCC diagnostic ignored "-Wsign-conversion"
```

Comment:
I think I'd rather still have these warnings show up, even for false positives on alpine/musl. I think most of the warnings are caused by NLA_HDRLEN being misdefined, so maybe an alternative would be to define a proper NLA_HDRLEN and _Static_assert it to be equal to the original NLA_HDRLEN. I don't want it to potentially block this review though, so I'm fine with getting some warnings on false positives.

Reply:
I used `static_assert` from `assert.h`, defined `_NLA_HDRLEN`, and cleaned up some others.  So now, just the one warning.

Reply:
And of course, had to change to `_Static_assert` :stuck_out_tongue: